file/writable_file_writer.cc

```diff
@@ -216,9 +216,17 @@ Status WritableFileWriter::Flush() {
   return s;
 }
 
+std::string WritableFileWriter::GetFileChecksum() {
+  if (checksum_generator_ != nullptr) {
+    return checksum_generator_->GetChecksum();
+  } else {
+    return kUnknownFileChecksum;
```

Comment:
I guess it's not a part of the PR but does it make more sense to call it `kNoFileChecksum`? Similarly for `kUnknownFileChecksumFuncName`

Reply:
@siying So change to kNoFileChecksum and kNoFileChecksumGenName ?

Reply:
Yes.

Reply:
@siying. I will change them in the next PR.